doc/go/capnp.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: none
+---
+<!doctype html>
+<html>
+  <!--
+    This page exists to set up an import path for the Go implementation; see:
+
+    * https://golang.org/cmd/go/#hdr-Remote_import_paths
+    * https://github.com/capnproto/go-capnproto2/issues/166
+  -->
+  <head>
+    <meta name="go-import" content="capnproto.org/go/capnp git https://github.com/capnproto/go-capnproto2" />
```

Comment:
You'll also want to add `go-source` to help direct godoc.org (and I think pkg.go.dev as well?) to show the data correctly.

```suggestion
    <meta name="go-import" content="capnproto.org/go/capnp git https://github.com/capnproto/go-capnproto2.git">
    <meta name="go-source" content="capnproto.org/go/capnp https://github.com/capnproto/go-capnproto2 https://github.com/capnproto/go-capnproto2/tree/master{/dir} https://github.com/capnproto/go-capnproto2/blob/master{/dir}/{file}#L{line}">
```

Reply:
Applied, though the `.git` suffix is not strictly necessary.